<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimondConfigDlg</class>
 <widget class="QWidget" name="SimondConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>simon can be set up to use a central server on the network. The model generation and recognition will then happen on the server, disburdening the clients.

Most users will leave this page to the default values. You can find more information on the topic in the simon manual.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbConnectAutomatically">
     <property name="text">
      <string>Connect automatically when simon starts</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbLocalInstallation">
     <property name="text">
      <string>Local installation (Default)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Use a local (private) recognition server (simond) installed and running on this computer.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbStartSimond">
         <property name="text">
          <string>Start the server automatically when simon starts</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbStopSimond">
         <property name="text">
          <string>Stop the server automatically when simon stops</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="rbRemoteInstallation">
     <property name="text">
      <string>Use central server on the network</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Server:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="leServer">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>114</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rbRemoteInstallation</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbRemoteInstallation</sender>
   <signal>toggled(bool)</signal>
   <receiver>leServer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLocalInstallation</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLocalInstallation</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbStopSimond</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLocalInstallation</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbStartSimond</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbStartSimond</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbStopSimond</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbStartSimond</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbStopSimond</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLocalInstallation</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbStartSimond</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLocalInstallation</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbStopSimond</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
